src/main/java/nextstep/fp/SumCondition.java

```diff
@@ -0,0 +1,6 @@
+package nextstep.fp;
+
+public interface SumCondition {
```

Comment:
```suggestion
@FunctionalInterface
public interface SumCondition {
```

현재 제공되는 메서드가 하나만 있으니 FunctionalInterface 애노테이션으로 제약을 걸어보는건 어떨까요?

참고) https://zzang9ha.tistory.com/303